docs/content/content-blocks.md

```diff
@@ -150,16 +180,99 @@ results in:
 Here's what's inside!
 ```
 
+(content/definition-lists)=
+
+## Definition Lists
+
+Definition lists are enabled by setting in your `_config.yml`:
+
+```yaml
+parse:
+  myst_extended_syntax: true
+```
+
+Definition lists utilise the [markdown-it-py deflist plugin](https://markdown-it-py.readthedocs.io/en/latest/plugins.html), which itself is based on the [Pandoc definition list specification](http://johnmacfarlane.net/pandoc/README.html#definition-lists).
```

Comment:
@chrisjsewell do we support any special styling in the theme `sphinx-book-theme`? I guess I am curious how we do modular additions to `themes` such as if we did include styling.

Reply:
See https://github.com/executablebooks/sphinx-book-theme/pull/191, although I'm open to improvements, since I literally added it a few days precisely to get this to work 😄

Reply:
basically this is the current SCSS:

```scss
        dl.simple {
            & dd {
                margin-left: 1.5em;
            }
            & dd:not(:last-child) {
                margin-bottom: 0px;
                p:last-child {
                    margin-bottom: 0px;
                }
            }
        }
```

Reply:
Cool. So essentially a theme needs to support all the elements -- and if they are present they get styled -- otherwise it doesn't get used. So this `SCSS` doesn't get added to the theme (via plugin style code injection) if definition lists are used.I don't see this as a problem -- just understanding how themes work with `modularity`.

Reply:
Yeh I guess basically the key to good theme/CSS modularity is having lots of classes on the HTML elements, so its easier to precisely specify things.